packages/in_app_purchase/in_app_purchase_ios/test/fakes/fake_ios_platform.dart

```diff
@@ -104,6 +106,20 @@ class FakeIOSPlatform {
         originalTransaction: null);
   }
 
+  SKPaymentTransactionWrapper createCanceledTransaction(
```

Comment:
nit: method name contains a typo:
```suggestion
  SKPaymentTransactionWrapper createCancelledTransaction(
```